wasm/bindings/TranslationModelBindings.cpp

```diff
@@ -6,17 +6,21 @@
 
 #include <emscripten/bind.h>
 
-#include "TranslationModel.h"
+#include "response.h"
+#include "service.h"
 
 using namespace emscripten;
 
+typedef marian::bergamot::Service TranslationModel;
+typedef marian::bergamot::Response TranslationResult;
+
 // Binding code
 EMSCRIPTEN_BINDINGS(translation_model) {
   class_<TranslationModel>("TranslationModel")
-    .constructor<std::string>()
-    .function("translate", &TranslationModel::translate)
-	  .function("isAlignmentSupported", &TranslationModel::isAlignmentSupported)
```

Comment:
This is [unused](https://github.com/mozilla-extensions/bergamot-browser-extension/search?q=isAlignmentSupported) at extension.

Reply:
Leave it in, implement it correctly